calico-policies/public-network-isolation/us-east/allow-public-services-pods.yaml

```diff
@@ -37,16 +37,15 @@ spec:
   - action: Allow
     destination:
       nets:
-      # IBM Cloud Monitoring with Sysdig: US East nets
-      - 169.47.136.237/32
-      - 169.47.35.174/32
-      - 169.55.109.114/32
+      # IBM Cloud Monitoring with Sysdig: us-east
+      # Taken from: https://cloud.ibm.com/docs/monitoring?topic=monitoring-endpoints
       - 169.60.112.74/32
-      - 169.61.109.59/32
-      - 169.61.65.234/32
+      - 169.55.109.114/32
       - 169.62.3.82/32
-      - 169.63.158.254/32
-      - 169.63.163.50/32
+      # IBM Cloud Monitoring with Sysdig: ca-tor subnets
```

Comment:
Why do us-east clusters need to access ca-tor sysdig IPs?  I don't see this mentioned in the docs specified here: https://cloud.ibm.com/docs/monitoring?topic=monitoring-endpoints

Reply:
@bradbehle , when deploying to single zone ca-tor, we are using us-east policies as mentioned in the note. That is the reason

```
NOTE: The policies in the ca-tor and br-sao directories are meant for use with the Toronto and São Paulo multizone locations. For the Toronto single zone location, use the policies in the us-east directory instead. For the São Paulo single zone location, use the policies in the us-south directory instead.
```